snap/snapcraft.yaml

```diff
@@ -63,6 +63,13 @@ description: |
 
   The DCGM snap does not currently include the performance testing tool `dcgmproftester` in order to limit the size of the snap.
 
+  **Cryptography**
+  ---
+
+  The snap installs the CUDA keyring, verifying its integrity using `curl` and SHA256 checksums.
```

Comment:
I think you should be a bit more specific here: we don't verify the integrity of the CUDA keyring, but rather the integrity of the CUDA keyring deb package (which contains the keyring)